src/platform/intel/cavs/lib/dma.c

```diff
@@ -109,7 +109,7 @@ static const struct dw_drv_plat_data dmac1 = {
 };
 
 /* DMA number of buffer periods */
-#define HDA_DMA_BUFFER_PERIOD_COUNT	2
+#define HDA_DMA_BUFFER_PERIOD_COUNT	4
```

Comment:
what problem does it fix or improve @kv2019i

Reply:
mono 8khz "doesn't transfer" without this. I think without this the buffer size is going under some dma transfer limit -> no data moving.

Reply:
@ranj063 @plbossart We can dig deeper before merging. This is now aligning with intel/ace/lib/dma.c (as I don't know full reasons why it is 4 there, but apparently that works better).

Reply:
Why @lgirdwood would a change in buffer sizes prevent races? Not following at all.